apps/web/src/store/modules/user/actions.ts

```diff
@@ -17,7 +19,7 @@ const getDomainOwnerInfo = async (ownerId: string): Promise<Partial<UserState>>
         backend: 'LOCAL',
         name: response.name,
         email: response.email,
-        language: response.language,
+        language: languages[response.language] ? response.language : 'en',
```

Comment:
description: This is the conversion code which makes work with both, 'ja' and 'jp'.